#Params for trajectory follower

step_size: 0.2   # seg
look_ahead: 0.15 # m
